.github/scripts/configure-runners.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For running the script, see go/templates-gitactions-script
+
+# Defaults
+NAME_SUFFIX="it"
+SIZE=3
+BASE_NAME="gitactions-runner"
+REPO_NAME="DataflowTemplates"
+REPO_OWNER="GoogleCloudPlatform"
+GH_RUNNER_VERSION="2.299.1"
+VERBOSE=0
+
+############################################################
+# Help                                                     #
+############################################################
+Help()
+{
+   # Display Help
+   echo "Configure github actions runners for running"
+   echo "integration and performance tests."
+   echo
+   echo "Syntax: configure-runners [-p <PROJECT>] [-a <SA_EMAIL>] [-t <GITHUB_TOKEN>] [-n|S|r|o|s|v|V|h]"
+   echo "options:"
+   echo "p     Set the name of the Google Cloud project to use."
+   echo "a     Set the service account email for the Google Cloud project."
+   echo "t     Set the token used to authenticate the runners with the repo."
```

Comment:
very nit but usually `[-arg <val>]` is used for optional arguments, and here those 3 are mandatory.

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html

> Arguments or option-arguments enclosed in the '[' and ']' notation are optional and can be omitted.